default.yml

```diff
@@ -538,21 +251,56 @@ Layout/EmptyLinesAroundModuleBody:
     - empty_lines_special
     - no_empty_lines
 
-Style/EmptyMethod:
-  EnforcedStyle: compact
+# Use empty lines between defs.
+Layout/EmptyLineBetweenDefs:
+  # If `true`, this parameter means that single line method definitions don't
+  # need an empty line between them.
+  AllowAdjacentOneLineDefs: false
+  # Can be array to specify minimum and maximum number of empty lines, e.g. [1, 2]
+  NumberOfEmptyLines: 1
+
+Layout/SpaceAroundEqualsInParameterDefault:
+  EnforcedStyle: space
```

Comment:
This is the only situation with `=` where I kind of prefer `no_space`, but if it's typical to have spaces in the Ruby community then okay.

Reply:
Yeah, I think it is, but if you want you can raise the point in slack